### PR TITLE
Improve stability and speed of E2E tests

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Cypress E2E CI Jobs

### Description
This PR fixes the stability and speed of the E2E job.

### Steps
- [x] Added / updated tests
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: src/components/editor-page/editor-pane/autocompletion/code-block.ts [hunk 1]
```diff
@@ -5,45 +5,103 @@
...
-        }
+/**
+ * Fetches the highlight js chunk.
+ * @return the retrieved highlight js api
+ */
+const loadHighlightJs = async (): Promise<highlightJsImport | null> => {
+  try {
+    return await import('../../../common/hljs/hljs')
+  } catch (error) {
+    console.error("can't load highlight js", error)
```
Comment (ErikMichelson) [inline src/components/editor-page/editor-pane/autocompletion/code-block.ts:23]: Maybe use the new notification system instead.

Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/autocompletion/code-block.ts:23]: Hm. Maybe. But I'm not sure how much information we should provide. Maybe just "System error occured while doing xyz. Check browser console and report the error if it occurs again." ?
Notifications are for end users.

Comment (ErikMichelson) [reply src/components/editor-page/editor-pane/autocompletion/code-block.ts:23]: Yes, it's just that end-users get notified that there's a problem and that they may seek help in our matrix chat.

Most people don't look into the browser console if something isn't working and therefore just writing to the console is like a silent failure (which is bad UX).

Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/autocompletion/code-block.ts:31]: done

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Pull request merged**